### PR TITLE
change first event format to support DST

### PR DESCRIPTION
**Pull request by chantastic**
* removes post date validation
* removes extraneous `asDate` function
* removes cli question about `timezone` an prompts for UTC (short-term but effective solution)
* updates all existing dates to UTC (no offset)

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -15,9 +15,11 @@ const isProduction = process.env.NODE_ENV === 'production';
 // See: https://www.11ty.dev/docs/dates/#setting-a-content-date-in-front-matter
 // Note: we make sure the parsed date is valid, then check the actual raw input format to make sure we have consistency
 
-const dateFormatRegex = /(date:)\s[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(\+|\-)[0-9]{2}:[0-9]{2}/gm;
-const isValidDate = (date, rawInputContent) =>
-	!Number.isNaN(Date.parse(date)) && rawInputContent.match(dateFormatRegex);
+// const dateFormatRegex = /(date:)\s[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]{3}Z/gm;
+const isValidDate = (date, rawInputContent) => {
```
Comment (BenDMyers) [inline .eleventy.js:19]: If this function returns `true`, can we nuke the function?

Comment (msutkowski) [reply .eleventy.js:19]: You'd still want to ensure that we're using a specific & consistent date format, so the regex should most likely just be updated.

Comment (chantastic) [reply .eleventy.js:19]: yeah. i left the shell, so we could fux it at a later time.

tbh, the fallback behavior is the same. if the yaml date can't be parsed, that page is not built.

so i'm kinda ok with removing it unless there was another virtue

Comment (chantastic) [reply .eleventy.js:19]: i was not able to get an updated regex to work. the date that is being returned locally is a completely different format now. includes a `000Z`

Comment (BenDMyers) [reply .eleventy.js:19]: Perhaps it's best to lean on the tools we have:

The [Inquirer.js README](https://github.com/SBoudrias/Inquirer.js/#prompts) lists two possible plugins for sourcing datetimes:

- [`inquirer-date-prompt`](https://github.com/haversnail/inquirer-date-prompt)
- [`inquirer-datepicker-prompt`](https://github.com/DerekTBrown/inquirer-datepicker-prompt)

Might be worth seeing if these plugins could make our lives easier.

Comment (BenDMyers) [reply .eleventy.js:19]: ☝🏻 Investigating these options is totally worth another PR, imho. We can merge this one in and spin up a new issue.

Comment (chantastic) [reply .eleventy.js:19]: How would everyone feel about this path forward?

* i remove the validation from the eleventy config completely
  * we know that pages without a proper date will fail to build in isolation — not impacting the site as a whole
* i copy the current date validation function for reference and open a new PR
  * we investigate adding the validation back in and the `inquirer-*` methods above in a follow-up PR

thoughts? additions? objections?

Comment (BenDMyers) [reply .eleventy.js:19]: You have my axe!

**Comment by chantastic** [comment]
This is merged, per our discussion.

/cc @BenDMyers

**Pull request merged**